### DISCUSSION TITLE
Error Authentication

### DISCUSSION THREAD

**Discussion by marouane-lhamidi** [keycloak/keycloak#20335] [Q&A]
Keycloak instance = Keycloak.getInstance("http://0.0.0.0:8080/", "myrealm", "user", "user_password","client_id", "client_secret"); TokenManager tokenmanager = instance.tokenManager(); String accessToken = tokenmanager.getAccessTokenString();
i want to return the token but it does not i don't no where problem is coming from? do you have an idea?

**Comment by danielFesenmeyer** [comment]
Probably you should try 127.0.0.1 instead of 0.0.0.0, because the latter is non-routable.

See https://www.howtogeek.com/225487/what-is-the-difference-between-127.0.0.1-and-0.0.0.0/

**Reply by danielFesenmeyer** [reply]
It that's not the problem, you should provide more details, like the returned http status and body.

**Comment by marouane-lhamidi** [comment]
I don’t know to be honest from where the problem but is working well from
where was the problem i don’t know anyway thank you for your response

Le lun. 15 mai 2023 à 13:56, Daniel Fesenmeyer ***@***.***> a
écrit :

> It that's not the problem, you should provide more details, like the
> returned http status and body.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/keycloak/keycloak/discussions/20335#discussioncomment-5904979>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARZSYUPCI2TGGOKHNS5JPQLXGIRYTANCNFSM6AAAAAAYAPSINI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 

*Elève ingénieur en Informatique et Réseaux option (MIAGE)*
*De L'École Marocaine des Sciences de l'Ingénieur*
*Université Hassan II de Casablanca, Maroc*
*Emails* : ***@***.*** ***@***.***> ,
***@***.***
***@***.***>*

*Téléphone  : (+212) 6 42 94 29 21*
